packages/matching-demo/src/algorithm/spreadMatcher.ts

```diff
@@ -0,0 +1,302 @@
+import { omit, cloneDeep } from 'lodash';
```

Comment:
This is copy-paste from algorith running in backend, with one more value returned from the function for debugging purposes